stores/testdata/contents.txt

```diff
@@ -1 +1 @@
-contents
\ No newline at end of file
+contents
```

Comment:
This is correct but causing a test to fail. Can you run `make unit` test and add the missing byte :)

Reply:
Sure thing!